ffi/blitbuffer.lua

```diff
@@ -973,7 +973,7 @@ end
 
 function BB_mt.__index:blitDefault(dest, dest_x, dest_y, offs_x, offs_y, width, height, setter, set_param)
     -- slow default variant:
-    local hook, mask, count = debug.gethook()
+    local hook, hmask, _ = debug.gethook()
```

Comment:
Bit late to comment I suppose, but to drop it you could also just ignore that third one fully.

Reply:
True, I do generally like the _ placeholder though (mostly for documentation purposes ;)).